### PR TITLE
smbios option description

### PR DESCRIPTION
**Pull request by aneustroev**
##### SUMMARY
More detailed description smbios option

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

community.general.proxmox_kvm

### CODE REVIEW

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution. It's better to use formatting, as in my suggestion - see https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation for details.

File: plugins/modules/proxmox_kvm.py [hunk 1]
```diff
@@ -415,6 +415,14 @@
...
     description:
       - Specifies SMBIOS type 1 fields.
+      - Comma separated, base64 encoded (optional) SMBIOS properties
+      - [base64=<1|0>] [,family=<Base64 encoded string>] 
+      - [,manufacturer=<Base64 encoded string>] 
+      - [,product=<Base64 encoded string>] 
+      - [,serial=<Base64 encoded string>] 
+      - [,sku=<Base64 encoded string>]
+      - [,uuid=<UUID>]
+      - [,version=<Base64 encoded string>]
```
[the remark this answers is not in the document]
Comment (aneustroev) [reply plugins/modules/proxmox_kvm.py:425]: Thank you, it looks better.

**Comment by felixfontein** [comment]
Can you please change this back to a list of paragraphs, instead of one single line of text? (Newlines inside paragraphs are ignored.)

To avoid the map error, you need to add quotes to the line that has a `:` at its end.

**Review by UnderGreen — APPROVED** [review]

**Comment by felixfontein** [comment]
@aneustroev thanks as wellf or this one!
@UnderGreen thank you again as well :)

**Pull request merged**